packages/entities/entities-shared/src/components/entity-base-config-card/ConfigCardItem.vue

```diff
@@ -293,6 +296,11 @@ const componentAttrsData = computed((): ComponentAttrsData => {
       }
   }
 })
+
+const textContent = ref<HTMLElement>()
+
+const { isTruncated } = composables.useTruncationDetector(textContent as Ref<HTMLElement>)
```

Comment:
question: @Leopoldthecoder does it seem odd to pass in a `ref` to the function instead of the underlying value?

Reply:
I think it's acceptable. We have other composables (e.g. `useFetcher`) that take a `MaybeRef` param. Besides in the `useTruncationDetector` function, we need to watch this `textContent` param.

Reply:
Wondering if that same maybe logic should be used here for safety

Reply:
Ah yeah I think it'd be nice. More unified with other composables.

Reply:
@filipgutica can you add the logic to handle if a user passes in a `ref` or just the HTML element? The composable should be able to handle either

Reply:
but if I watch the `unRefed` value does that break the reactivity?

Reply:
done

Reply:
Yes. If you explicitly want a `Ref` I would add block comments above the `export default function useTruncationDetector` line so that it's shown during code hinting; something like:

```ts
/**
 * Detect when blah blah blah
 * @param Ref<HTMLElement> textContent The description of this param... 
 * @returns ... the thing
 */
export default function useTruncationDetector(textContent: Ref<HTMLElement>) {
```

Reply:
👍